### ISSUE TITLE
Full exp is being rewarded in level-restricted Promyvion areas

### ISSUE THREAD

**Issue by isotor** [DarkstarProject/darkstar#5874]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 30190328_2

**_Source Branch_** (master/stable) **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

The correct behavior is outlined [here](https://ffxiclopedia.fandom.com/wiki/Promyvion_Guide?oldid=70377):

> Experience earned in a level-capped zone is either... 
> 1) You earn the xp you'd get if your class were at it's full level. 
> 2) You earn 1/2 the xp of your capped job. 
> ... whichever is greater. If you're on a lv31 main capped down to 30, you'll get lots of xp and can do xp chains. If you're on a 60 main capped down to 30, you'll get 1/2 the xp a lv30 would earn, and you can't do xp chains. 

__tl;dr:__ exp should be calculated based on your "uncapped" job level, to a minimum of 50% of the exp a level 30 character would earm _(or 40 for Vahzl)_, and you can't do exp chains.

**Comment by TeoTwawki** [comment]
I'm on the fence about implementing this directly into the restriction or as a setting. that was added in to curb ppl using those restricted zones to reach lv cap, and then the restriction was later done away with entirely anyway.
